tools/ci_build/github/azure-pipelines/templates/android-java-api-aar-test.yml

```diff
@@ -9,6 +9,12 @@ parameters:
   type: string
   default: ''
 
+- name: packageName
+  # now we can build onnxruntime-android or onnxruntime-mobile for Android, need specify it here
+  displayName: Package Name
+  type: string
```

Comment:
FYI, it's possible to limit the options with "values". there's an example here:
[https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops#parameter-data-types](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops#parameter-data-types)